src/components/NetworkConnectivity.js

```diff
@@ -151,7 +151,7 @@ class NetworkConnectivity extends React.PureComponent<Props, State> {
     this.checkInternet();
   };
 
-  handleConnectivityChange = (isConnected: boolean) => {
+  handleConnectivityChange = ({ isConnected }) => {
```

Comment:
Same as above

Reply:
This was caused a problem I don't know why!